src/csharp/Microsoft.Spark.E2ETest/BridgeTests/JvmBridgeTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Spark.Sql;
+using Xunit;
+
+namespace Microsoft.Spark.E2ETest.BridgeTests
+{
+    [Collection("Spark E2E Tests")]
+    public class JvmBridgeTests
+    {
+        [Fact]
+        public void TestSparkExceptionReturned()
+        {
+            using var session = SparkSession.GetDefaultSession();
+            try
+            {
+                session.Sql("THROW!!!");
+            }
+            catch (Exception sparkEx)
+            {
+                Assert.NotNull(sparkEx.InnerException);
```

Comment:
Can we test more such as inner exception type, etc.?

Reply:
Added type check
Added test of inner exception message